### ISSUE TITLE
Wrong description for type of $device

### ISSUE THREAD

**Issue by zaycker** [matomo-org/device-detector#5934]
```
/**
 * Holds the device type after parsing the UA
 * @var string
 */
protected $device = null;
```

but

```
/**
* Returns the device type extracted from the parsed UA
*
* @see DeviceParserAbstract::$deviceTypes for available device types
*
* @return int|null
*/
public function getDevice()
{
    return $this->device;
}
```

**Comment by zaycker** [comment]
found this inconsistency in a process of migration to Go

**Comment by sgiehl** [comment]
fixed in #5935
